UOP1_Project/Assets/Scripts/Systems/Settings/SettingsSystemGeneralComponent.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class SettingsSystemGeneralComponent : SettingsSystemSubComponents
+{
+    [SerializeField] TMP_Dropdown languageDropdown;
+
+    SettingsSystem settingsSystem;
+    public LanguageSetting Language { get; private set; }
+    
+    public enum LanguageSetting
+    {
+        English,
+        German,
+        //TODO: which languages are going to be supported?
```

Comment:
Why did you ask this question in code? Please create a new thread here https://forum.unity.com/forums/open-projects.531/

Reply:
This was a reminder for myself actually. And as its not that an important question I didn't ask it in the forums for now.

Reply:
Imho language support is very important.

Reply:
It's unofficially planned: https://forum.unity.com/threads/localization-system.982776/